src/node/chainstatemanager_args.cpp

```diff
@@ -34,6 +36,15 @@ std::optional<bilingual_str> ApplyArgsManOptions(const ArgsManager& args, Chains
 
     if (auto value{args.GetIntArg("-maxtipage")}) opts.max_tip_age = std::chrono::seconds{*value};
 
+    int par_value = args.GetIntArg("-par", DEFAULT_SCRIPTCHECK_THREADS);
+    if (par_value <= 0) {
+        // -par=0 means autodetect (number of cores - 1 script threads)
+        // -par=-n means "leave n cores free" (number of cores - n - 1 script threads)
+        par_value += GetNumCores();
+    }
+    opts.worker_threads_num = std::clamp(par_value - 1, 0, MAX_SCRIPTCHECK_THREADS);
```

Comment:
Missing the comment explaining the `- 1`

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/26762#issuecomment-1408502815).